pubsub/pubsub.go

```diff
@@ -29,9 +29,36 @@ type PubSub interface {
 	Close() error
 }
 
+// BulkPublisher is the interface defining BulkPublish definition for message buses
+type BulkPublisher interface {
+	// BulkPublish is the method to publish multiple messages to a topic in a bulk fashion.
```

Comment:
We can remove this. Example from stdlib https://cs.opensource.google/go/go/+/refs/tags/go1.19.1:src/io/io.go;l=96